### DISCUSSION TITLE
[Feature] Documentation

### DISCUSSION THREAD

**Discussion by mprajescu** [mediacms-io/mediacms#607] [Feature Requests]
A place to access all the documentation is a must these days for open source software. 

Is it possible to implement a Wiki JS on the mediacms.io domain such as docs.mediacms.io? 

I would like admin access to be able to create documentation

**Comment by mgogoulos** [comment]
working on it, we have a prototype with Gatsby/Docz that will be made available soon! Also thanks for requesting to contribute to it

**Comment by sthierolf** [comment]
Hi, I'm interested in a documentation too, especially how to set up scalable transcoding with remote workers.
I'm thinking on a setup where a VM with the frontend is running on phys. node 1 and a VM for transcoding on phys. node 2 to distribute the load on between VM's and their physical nodes.

I'm also interested how to use external storage systems, for example via NFS or iSCSI for the video content/files.

**Comment by mprajescu** [comment]
@mgogoulos any updates on the documentation location? Almost a year has passed and I understand everyone is working on more important and critical bugs, however, people are excited to contribute and add value to this open source project, such as creating documentation, translations, etc. 

I checked the status after 10 months and it seems a little bit better. I would like to point out that a FAQ page and a documentation page would be in the benefit of the project.

**Comment by mgogoulos** [comment]
Thanks for raising this! Yes, there has to be progress with the docs on this repository, and it's better than no docs at all, but long term there has to be another solution. 

I'm experimenting with Docusaurus, as @shubhank-saxena has suggested and I am very close to have a first version of the docs (basically just copying anything at docs/) . Most probably this should be uploaded in a few days on docs.mediacms.io or something similar, and this will also be another repository so that you can fork and work on it. 

But I'm very open to suggestions on this or other systems.
